src/app/contextBroker/contextBroker.cpp

```diff
@@ -1182,6 +1182,13 @@ void orionExit(int code, const std::string& reason)
 */
```

Comment:
Usually we don't include CNR entry for de-leaking PR, as the PR is supposed to fix the leaks introduced in the last version. However, due to:

> Thanks to this modification, 7-8 new (old :-)) leaks, until now undetected, have been found and fixed.

in this case I think we should include an entry.